staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -631,14 +631,14 @@ type KubeletConfiguration struct {
 	ShowHiddenMetricsForVersion string `json:"showHiddenMetricsForVersion,omitempty"`
 	// systemReservedCgroup helps the kubelet identify absolute name of top level CGroup used
 	// to enforce `systemReserved` compute resource reservation for OS system daemons.
-	// Refer to [Node Allocatable](https://git.k8s.io/community/contributors/design-proposals/node/node-allocatable.md)
```

Comment:
link all the node-allocatable fields to https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable instead?